s3/s3.js

```diff
@@ -0,0 +1,68 @@
+const AWS = require('aws-sdk');
+const { logger } = require('../services');
+
+let s3 = new AWS.S3({ endpoint: 'http://localstack:4572', region: 'us-west-1', s3ForcePathStyle: true });
```

Comment:
We'll need to make endpoint reliant on `process.env` variables since production won't use the localstack endpoints.

Reply:
👍 Updated to include an `env.example` and also included the SQS endpoints there